src/models.py

```diff
@@ -0,0 +1,47 @@
+import uuid
+from sqlalchemy import Boolean, Column, ForeignKey, String
+from sqlalchemy.dialects.postgresql import UUID
```

Comment:
import sqalchemy as sa and access its content through dot. It is better than importing requirements directly. The same for UUID. Main reason - there are a lot of same named stuff in built in libraries of python. Sometimes you can accidently have collision and your code will break ungracefully